app/policies/activity_policy.rb

```diff
@@ -69,7 +69,7 @@ def redact_from_iati?
   end
 
   def update_linked_activity?
-    return unless record.is_ispf_funded?
+    return unless ROLLOUT.active?(:activity_linking) && record.is_ispf_funded?
```

Comment:
I wonder if we want to flip this to match the other feature flags.

`return if ROLLOUT.active?(:hide_activity_linking)`

Reply:
Worth saying that if the flag is absent for any reason, that the safer option is to keep this as it is `ROLLOUT.active?(:activity_linking)`

Reply:
I think `active?(FEATURE_NAME)` is more readable too as we're talking about the feature being active, rather than hiding it being active 🤯